projects/laji/src/app/shared/service/form.service.ts

```diff
@@ -37,48 +39,50 @@ export class FormService {
     private http: HttpClient,
     private userService: UserService,
     private translate: TranslateService
-  ) {}
+  ) {
+    this.translate.onLangChange.subscribe(this.setLang);
```

Comment:
The return type of this is not the language so it should be more like `(event) => this.setLang(event.lang)`